ApolloExtra.lua

```diff
@@ -2308,103 +2332,134 @@ OlympusLootData.ApolloUpgrade = {
+					Text = "Aphrodite and I are here to help. We came to a conclusion. Without our help, you won't escape. Therefore... " },
+				{ Cue = "/VO/Aphrodite_0230",
+					PortraitExitWait = 0.35,
+					PreLineFunctionName = "BoonInteractPresentation", PreLineWait = 0.5,
+					Emote = "PortraitEmoteAffection",
+					EndSound = "/Leftovers/World Sounds/MapZoomInShort",
+					Speaker = "NPC_Aphrodite_01", Portrait = "Portrait_Aphrodite_Default_01",
+					Text = "This is for you, little godling! Oh my me! I can't wait until we are all here together!"
+				},
 			},
 			ApolloWithArtemis01 =
 			{
                 Name = "ApolloWithArtemis01",
 				PlayOnce = true,
 				PreEventFunctionName = "BoonInteractPresentation", PreEventFunctionArgs = { PickupWait = 1.0, },
-				HasTraitNameInRoom = "HeartsickCritDamageTrait",
+				HasTraitNameInRoom = "FamedDuetTrait",
 				RequiredTextLines = { "ApolloGift01", "ArtemisGift01" },
-				{ Cue = "/VO/Apollo_0185",
+				{ Cue = "/VO/Apollo_0046",
 					Emote = "PortraitEmoteAffection",
 					StartSound = "/Leftovers/World Sounds/MapZoomInShort",
-					Text = "Oh dearest! I have brought you keen-eyed Artemis, who, in her focus on the hunt, has not developed social graces suitable enough to come to you herself!" },
-				{ Cue = "/VO/Artemis_0239",
+					Text = "I brought my sister with me this time, Zagzag! With our help, you might stand a chance against your father. Go Zagzag!" },
+				{ Cue = "/VO/Artemis_0250",
 					PortraitExitWait = 0.35,
 					PreLineFunctionName = "BoonInteractPresentation", PreLineWait = 0.5,
 					StartSound = "/SFX/ArtemisBoonArrow",
 					EndSound = "/Leftovers/World Sounds/MapZoomInShort",
 					Emote = "PortraitEmoteDepressed",
 					Speaker = "NPC_Artemis_01", Portrait = "Portrait_Artemis_Default_01",
-					Text = "Social graces, she says! Going about like {#DialogueItalicFormat}that{#PreviousFormat}! I'm sure you know that I can find you when it's necessary, Zagreus. Though, how to get away from Apollo, I have no idea."
+					Text = "Are you seriously calling him that... I am so sorry Zagreus. I hope this can make him a little more to bearable..."
 				},
 			},
 			ApolloWithDionysus01 =
 			{
                 Name = "ApolloWithDionysus01",
 				PlayOnce = true,
 				PreEventFunctionName = "BoonInteractPresentation", PreEventFunctionArgs = { PickupWait = 1.0, },
-				HasTraitNameInRoom = "DionysusApolloStackIncreaseTrait",
-				{ Cue = "/VO/Apollo_0211",
+				HasTraitNameInRoom = "MaxHangoverTrait",
+				{ Cue = "/VO/Apollo_0047",
 					StartSound = "/Leftovers/World Sounds/MapZoomInShort",
-					Text = "My powers, dearest, oft go hand-in-hand with those of Dionysus, with his power to persuade the mind to find such beauty where it isn't, really!" },
-				{ Cue = "/VO/Dionysus_0211",
+					Text = "Woah this wine is exquisite, Brother! I can't believe how well it goes with the cheese. The taste of both is just... Out of this world!" },
```

Comment:
"Woah, this wine is exquisite, Brother! I can't believe how well it goes with the cheese. The taste of both is just... out of this world!" },